qiskit/qobj/qasm_qobj.py

```diff
@@ -0,0 +1,482 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=invalid-name
+
+"""Module providing definitions of QASM Qobj classes."""
+
+import os
+from types import SimpleNamespace
+
+import json
+import fastjsonschema
+
+
+path_part = 'schemas/qobj_schema.json'
```

Comment:
Are the schemas included in the setuptools installation as part of package_data in setup.py?

Reply:
Yes, they're included as part of the package data via the manifest.in file: https://github.com/Qiskit/qiskit-terra/blob/master/MANIFEST.in#L3